### PR TITLE
Reconnect After Stream Loss

### PR DESCRIPTION
**Pull request by DukeyAir**
Reconnect after Stream is Lost/Disconnected i.e. battery swapping on kwad. No need to reopen app now.

### CODE REVIEW

File: app/src/main/java/com/fpvout/digiview/MainActivity.java [hunk 1]
```diff
@@ -113,6 +117,20 @@ private void connect(){
         mUsbMaskConnection.setUsbDevice(usbManager.openDevice(usbDevice), usbDevice);
         mVideoReader.setUsbMaskConnection(mUsbMaskConnection);
         mVideoReader.start();
+        mVideoReader.mPlayer.addListener(new ExoPlayer.EventListener() {
```
Comment (jlucidar) [inline app/src/main/java/com/fpvout/digiview/MainActivity.java]: can you merge this with the already declared event listener in VideoReaderExoplayer class (in start function / line 64) please ?
Better keep event handling on the player at the same place :)

**Review by jlucidar — CHANGES_REQUESTED** [review]

**Comment by dukey32123** [comment]
Done!

File: app/src/main/java/com/fpvout/digiview/MainActivity.java (hunk 1 above)
Comment (DukeyAir) [reply app/src/main/java/com/fpvout/digiview/MainActivity.java]: Please hold off the PR, i will post a massive rework today, just gotta get some sleep first :D

**Comment by jlucidar** [comment]
Sure. Let me know when you want me to review and merge then :)

**Comment by DukeyAir** [comment]
Closing this one since PR #32 already has it implemented in a cleaner way

**Pull request closed without merging**